README.md

```diff
@@ -24,24 +24,28 @@
 # 📖 [Hello-Microverse] <a name="about-project"></a>
```

Comment:
- [ ] Kindly update the headline of this section , it should be just like current project name instead of the "Hello-Microverse".